README.md

```diff
@@ -2,39 +2,60 @@
 
 # OCaml - Continuous benchmarks
 
-Prototype for running predictable, IO-bound benchmarks in an ocurrent pipeline. This is *work in progress*.
-If you want to be on the allowlist for running benchmarks for your repository, please contact @gs0510, or you can open an issue.
+Prototype for running predictable, IO-bound benchmarks in an ocurrent pipeline. This is *work in progress*. If you want to be on the allowlist for running benchmarks for your repository, please contact @gs0510, ask around on [slack](https://example.com)<!--- FIXME: add real slack link if publishing it isn't an issue -->, or [open an issue](https://github.com/ocurrent/current-bench/issues/new).
 
-## Enroll your project
+If you're here just to enroll your repo to see nice graphs, you're in the right place, you can jump to the next section.
+If you want to run this project locally on a tuned machine, see [TUNING.md](TUNING.md).
+If you're here to contribute to improving current-bench, see [HACKING.md](HACKING.md)
 
-If you want to enroll your repository or setup this benchmark repository for your repository, we make the following assumptions.
+## The github application
+- First, you'll need to make sure you have admin access to the organization that hosts your repo. If it's a private repo that just means your own account.
+- Then go to the app's page on the [Github Marketplace](https://github.com/marketplace/ocaml-benchmarks), scroll down and click on "Install it for free". 
+- Choose "only select repositories" and select the repo you want benchmarked. If this is a personal project, you might have to deal with 2FA there.
 
-1. There is a `make bench` target which can run the benchmarks.
-2. The benchmarks result is a JSON object of the following format:
+Done!
 
-```bash
+## Tell us about it
+Since current-bench is in an experimental stage, users have to be approved manually. Ask us to approve you or your org. Someone on our team will notify you when it's been taken care of.
+
+Now, you should be able to see your repo in the dropdown menu at https://autumn.ocamllabs.io
+
+## Ensuring we can run your benchmarks
+To be able to run your benchmarks, current-bench assumes certain things about your repo:
+- You have a `.opam` file at the root of your project specifying your dependencies and all the usual metadata.
+- Your benchmarks can be run by using `make bench` at the root of your project. If you don't have one, you'll need a `Makefile`. (For example, see [this simple Makefile](https://github.com/example-ocaml-org/my-ocaml-project/blob/main/Makefile))
```

Comment:
This is great! Thanks for creating the example or with the repo, and everything! :100:

Reply:
Thanks! I plan on fledging it up a bit, with a simple dune file and everything